src/DynamoCore/BuiltInAndOperators/Operators.xml

```diff
@@ -6,106 +6,119 @@
     <members>
         <member name="%add">
             <summary>
-                Adds x to y.
+                Returns addition of x and y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer value, double value or string</param>
+            <param name="y">Integer value, double value or string</param>
+            <returns name="var">The sum of two input numbers, or the concatenation of two strings</returns>
             <search>add, plus, addition, +</search>
         </member>
 		<member name="%sub">
             <summary>
-                x subtract y.
+                Returns subtraction of x minus y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="number">Result of the y input subtracted from the x input</returns>
             <search>sub, subtract, difference, -</search>
         </member>		
 		<member name="%mul">
             <summary>
-                Multiplies x by y.
+                Returns multiplication of x times y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="number">The product of the two input numbers</returns>
             <search>mul, multiply, product, times, *, x</search>
         </member>
 		<member name="%div">
             <summary>
-                Divides x by y.
+                Returns division of x over y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="double">Result of division</returns>
             <search>div, divide, product</search>
         </member>
 		<member name="%eq">
             <summary>
-                Equal x to y?
+                Compares if x and y are equal
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Basic type object to compare (e.g. bool, int, double, string, etc.)</param>
+            <param name="y">Basic type object to compare (e.g. bool, int, double, string, etc.)</param>
+            <returns name="bool">Result of equality check</returns>
             <search>eq, equal, =</search>
         </member>
 		<member name="%ge">
             <summary>
-                x greater or equals y?
+                Compares if x is greater or equals to y
```

Comment:
NIT: The correct form should be `greater or equal to` (drop the `s` on `equals`)

Reply:
Addressed in 0e11365